troubleshooting/tools/tracevol.py

```diff
@@ -125,27 +146,42 @@ def format_table(arg, pvc_data, table):
         table.add_row([pvcname, pvname, "", False,
                        False, False])
         return
-    # check image details present rados omap
-    pv_present, uuid_present = validate_volume_in_rados(
-        arg, image_id, pvname, pool_name)
-    image_in_cluster = check_image_in_cluster(arg, image_id, pool_name)
+    # check image/subvolume details present rados omap
+    pv_present, uuid_present = validate_volume_in_rados(arg,
+                                                        image_id,
+                                                        pvname,
+                                                        pool_name,
+                                                        isRBD)
+    present_in_cluster = False
+    if isRBD:
+        present_in_cluster = check_image_in_cluster(arg, image_id, pool_name)
+    else:
+        subVolName = "csi-vol-%s" % image_id
+        present_in_cluster = check_subvol_in_cluster(arg, subVolName)
     image_name = "csi-vol-%s" % image_id
```

Comment:
csi-vol can be overridden with storageclass argument.

Reply:
Right, the `"csi-vol"` prefix can be configured, so it needs to be dynamic. But that is something that needs to change in this Python script in any case, and is not related to this enhancement.

Reply:
Yeah, actually there are couple of similar such things in the existing script which are hard coded as of now and will not work. I can try to list those and will fix in a separate PR.

Reply:
Created https://github.com/ceph/ceph-csi/issues/1143 for this.